tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -195,6 +202,34 @@ public static function clearCacheAfterFeature()
         self::clearCache();
     }
 
+    /**
+     * @BeforeScenario @reset-context-before-scenario
+     */
+    public static function resetContextBeforeScenario()
```

Comment:
```suggestion
    public static function mockContextBeforeScenario()
```
Same naming as in `ContextFeatureContext` which represents better what it actually does